camel/retrievers/vector_retriever.py

```diff
@@ -130,7 +139,7 @@ def query(  # type: ignore
         # format the results
         formatted_results = []
         for result in query_results:
-            if (result.similarity >= similarity_threshold
+            if (result.similarity >= self.similarity_threshold
```

Comment:
It looks like adding self.similarity_threshold as default while accepting the threshold as an argument is a better solution. Why do we not have a threshold in the bm25 one?

Reply:
there's no fixed interval for bm25 score